continuous_integration/recipe/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version=environ.get('CONDA_PY', 36) %}
+
+
+package:
+  name: dask-core
+  version: {{ version }}
```

Comment:
I'm curious what this version looks like

Reply:
As an example, a build from today would be version `2021.12.0a211217`